File: ClassLibrary1/Digits.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClassLibrary1
{
    public class Digits : Class1
    {
        public Digits(string v) : base(v) { }

        public override string Replace()
        {
            string res = "";
            foreach (char c in value)
            {
                if (c == '3')
                    res += "11";
                else
                    res += c;
            }
            return res;
        }
    }
}

File: StaticLib1/Digits.cpp
#include "pch.h"
#include "Digits.h"
Digits::Digits(const string& v) : ryad(v) {}

string Digits::replace() const
{
	string res;
	for (int i = 0; i < value.length(); i++)
	{
		if (value[i] == '3')
		{
			res += "11";
		}
		else
			res += value[i];
	}
	return res;
}